packages/gdb.rb

```diff
@@ -50,6 +40,10 @@ class Gdb < Autotools
 
   conflicts_ok # binutils conflicts
 
+  def self.patch
+    system 'filefix'
+  end
+
   def self.build
```

Comment:
Why isn't this section using `configure_options` instead?

Reply:
Good call. Fixed.